producers/http/handlers.go

```diff
@@ -111,16 +111,10 @@ func containerAppHandler(p *producerImpl) http.HandlerFunc {
 			return
 		}
 
-		var combinedMetrics producers.MetricsMessage
```

Comment:
See previous feedback left on commit https://github.com/dcos/dcos-metrics/pull/100/commits/9f44bc0cc9241023c7a39a9bcf62f693c8773b5c which this one overrides.